src/transaction/log_manager.c

```diff
@@ -1729,6 +1747,19 @@ log_final (THREAD_ENTRY * thread_p)
 
   save_tran_index = LOG_FIND_THREAD_TRAN_INDEX (thread_p);
 
+  // mark and persist that transaction server with remote storage has been correctly closed
+  if (is_tran_server_with_remote_storage ())
```

Comment:
Added section here because, on a TSRS, the functions, early-outs upon ~1770:
```
if (log_Gl.append.vdes == NULL_VOLDES)
```
It might be needed/better to add a TSRS-specific `log_final` function and fork in the caller.

Reply:
Added [LETS-226](http://jira.cubrid.org/browse/LETS-226) for above comment.